### PR TITLE
Remove openvpn

### PR DESCRIPTION
**Pull request by AmitPhulera**
##### Announce New Release

- [ ] After merging, I will follow [these instructions](https://confluence.dimagi.com/display/saas/Announcing+changes+affecting+third+parties#Announcingchangesaffectingthirdparties-announcing)
to announce a new commcare-cloud release.

### CODE REVIEW

File: src/commcare_cloud/terraform/README.md [hunk 1]
```diff
@@ -64,163 +64,3 @@ You can run terraform with
 ```
 cchq <env> terraform apply
 ```
```
Comment (AmitPhulera) [inline src/commcare_cloud/terraform/README.md:66]: @dannyroberts Should we add details about setting up SSM here or is it automatically configured when a new account is created?

**Review by millerdev — APPROVED** [review]

File: src/commcare_cloud/terraform/README.md (hunk 1 above)
Comment (dannyroberts) [reply src/commcare_cloud/terraform/README.md:66]: Great question. That is indeed set up manually-ish in the console, but may also be partially confiured by aws control tower. I don't think we need to proactively document, but may be worth it next time we happen to run through it. Feel free to offline if you want to explore more

**Review by dannyroberts — APPROVED** [review]

**Comment by dannyroberts** [comment]
Confirm `terraform plan` is empty?

**Comment by AmitPhulera** [comment]
Verified that no changes related to `openvpn` showing up in India and Prod.

**Pull request merged**